main.go

```diff
@@ -28,8 +29,15 @@ func main() {
 		log.Panicln(re)
 	}
 
+	var timeout int
```

Comment:
@mdastous Je te suggère ceci :
```suggestion
	var timeout int = 5  # Valeur par défaut à 5
	ti, lu := os.LookupEnv("HTTP_TIMEOUT")
        # Si on a une variable d'environnement ...
	if ( lu ) {
	    timeout, cv := strconv.Atoi(ti)
	    # ...mais qu'on est pas capable de parser la variable
	    if ( cv != nil ) {
	        # Fail here directly
	        log.Panicln("Could not convert HTTP_TIMEOUT environment variable to int")
	    }
	}
	#else {
	#    Pas besoin de ce block, HTTP_TIMEOUT n'existe pas donc on fallback sur timeout = 5 (qui est déjà set par défaut)
	#}
	# life goes on avec timeout =5!
```